src/server.ts

```diff
@@ -35,7 +35,10 @@ export class Server {
         return new Promise((resolve, reject) => {
             this.serverProtocol().then(() => {
                 let host = this.options.host || 'localhost';
-                Log.success(`Running at ${host} on port ${this.options.port}`);
+                let portRegex = /([0-9]{2,5})[\/]?$/;
```

Comment:
Can you make this line:

```typescript
let port = this.getPort()
```

And define a new method named `getPort()` that does this in one place? Will clean up line 100.